src/executables/csql.c

```diff
@@ -1862,7 +1862,8 @@ csql_execute_statements (const CSQL_ARGUMENT * csql_arg, int type, const void *s
       if (session->statements != NULL)
 	{
 	  statement = session->statements[num_stmts];
-	  if (statement != NULL && strlen (statement->sql_user_text) >= statement->sql_user_text_len)
+	  if (statement != NULL && statement->sql_user_text != NULL
```

Comment:
In CUBRID, we code this type with the following style as well (not important):
` if (statement  && statement->sql_user_text`

Reply:
Thank you, i will change code as your recommend.